examples/12_span_write.py

```diff
@@ -0,0 +1,35 @@
+import openpmd_api as io
+import numpy as np
+
+
+def span_write(filename):
+    series = io.Series(filename, io.Access_Type.create)
+
+    datatype = np.dtype("double")
+    length = 10
+    extent = [10]
+    dataset = io.Dataset(datatype, extent)
+
+    iterations = series.write_iterations()
+    for i in range(10):
```

Comment:
Maybe another number than `10` to avoid confusion:
```suggestion
    for i in range(12):
```